### DISCUSSION TITLE
Jamulus iOS Beta is now available on TestFlight

### DISCUSSION THREAD

**Discussion by ann0see** [jamulussoftware/jamulus#3448] [Announcements]
We have published a Beta of Jamulus for iOS on TestFlight. This should NOT be considered as anywhere near finished.

There are still many GUI bugs which can be resolved by rotating the screen. We are aware of those. They are already logged in https://github.com/jamulussoftware/jamulus/issues/3406. Please do not report them.
Any other issues found with this version should be reported on GitHub as an issue and not via the inbuilt Beta feedback from iOS.

This version is based on the current version from main with slight adaptations on the Info.plist file ([Download modified Info.plist](https://github.com/user-attachments/files/18247306/Info.plist.txt)) to give an explanation for camera usage (which Jamulus doesn't need and needs more investigation) and an encryption waiver. 

Join the beta here: https://testflight.apple.com/join/xZRwhf63

**Comment by admin** [comment]
Please update/re-enroll as tester

**Comment by ann0see** [comment]
There's a Qt5 build on TestFlight which should be stable.
